### ISSUE TITLE
Links vary in destination making it hard to parse

### ISSUE THREAD

**Issue by mahtoid** [Squad-Wiki/squad-wiki-pipeline-map-data#86]
When a layer has no lanes (or one) then it displays in the pathing of: `capturePoints -> clusters -> links[] -> name, nodeA, nodeB`
However, when there are multiple lanes then it displays in the pathing of: `capturePoints -> lanes[] -> name, links[]`

It would be great if it was in the same place constantly. That way it'll be easy to parse over instead of having to care about two different places to do magic.

**Comment by Shanomac99** [comment]
Reworked lanes object. Object now functions as the following:

`capturePoints -> lanes -> links`
`capturePoints -> lanes -> laneObjects`

Changed `lanes` from array into object.
Moved lanes info from `lanes` into `laneObjects` while converting to an object to match upcoming change with clusters.
Added array of lane names `listOfLanes` into `lanes` to allow easier parsing of new `laneObjects` object.
